Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -121,7 +117,7 @@ extension ReminderDetailEditDataSource: UITableViewDataSource {
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         
         guard let section = ReminderSection(rawValue: section) else {
-            return UITableViewCell()
+            return ""
         }
         
         return section.displayText
```

Comment:
Just a suggestion:
Sometimes when you have to return ( or use ) a value when the optional unwrapping returns nil. You can use the default operator `??` .. Instead of doing 
```
guard let unwrappedValue = someOptional else {
    return defaultValue
}
return unwrappedValue
```
you can do this
```
return someOptional ?? defaultValue
```

Reply:
Understood